src/lang/en-US.ts

```diff
@@ -930,6 +930,14 @@ const Workflows: ResolvedIntlConfig['messages'] = {
     'workflows.autoDiscovery.label.addNewBindings': `Automatically add new collections`,
     'workflows.autoDiscovery.label.evolveIncompatibleCollection': `Breaking changes re-version collections`,
 
+    'workflows.sourceCapture.header': `Source Capture`,
+    'workflows.sourceCapture.cta': `link capture`,
+    'workflows.sourceCapture.cta.edit': `update capture`,
+    'workflows.sourceCapture.selected.none': `no source capture`,
+    'workflows.sourceCapture.optin.message': `Below you can select a source capture to link to your materialization. This ensures as new collections are discovered on the capture, they will automatically be added to the materialization's bindings.`,
+    'workflows.sourceCapture.optin.message2': `If you remove source capture, the bindings that were added are not removed.`,
```

Comment:
Should the article _the_ be inserted between _remove_ and _source_ (i.e., _If you remove the source capture, ..._)?

Reply:
Yeah... I had it that way and wasn't sure if it was needed. Been trying to make the content as short as possible. I'll add it back.